diff --git i/src/input.c w/src/input.c
index 36128e1..15a1f28 100644
--- i/src/input.c
+++ w/src/input.c
@@ -320,7 +320,7 @@ void _glfwInputChar(_GLFWwindow* window, uint32_t codepoint, int mods, GLFWbool
     if (!window->lockKeyMods)
         mods &= ~(GLFW_MOD_CAPS_LOCK | GLFW_MOD_NUM_LOCK);
 
-    if (window->callbacks.charmods)
+    if (window->callbacks.charmods && !(mods & GLFW_MOD_CONTROL))
         window->callbacks.charmods((GLFWwindow*) window, codepoint, mods);
