diff --git a/src/main.rs b/src/main.rs
index 0d70f89..1fcfb89 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -72,15 +72,13 @@ fn print_system_info(fields: &Fields) {
     let blue = COLORS.blue;
     let reset = COLORS.reset;
     let system_info = format!("
-    {cyan}     ▟█▖    {blue}▝█▙ ▗█▛          {user_info} ~{reset}
-    {cyan}  ▗▄▄▟██▄▄▄▄▄{blue}▝█▙█▛  {cyan}▖        {cyan}  {blue}System{reset}        {os_name}
-    {cyan}  ▀▀▀▀▀▀▀▀▀▀▀▘{blue}▝██  {cyan}▟█▖       {cyan}  {blue}Kernel{reset}        {kernel_version}
-    {blue}     ▟█▛       {blue}▝█▘{cyan}▟█▛        {cyan}  {blue}Shell{reset}         {shell}
-    {blue}▟█████▛          {cyan}▟█████▛     {cyan}  {blue}Uptime{reset}        {uptime}
-    {blue}   ▟█▛{cyan}▗█▖       {cyan}▟█▛          {cyan}  {blue}Desktop{reset}       {desktop}
-    {blue}  ▝█▛  {cyan}██▖{blue}▗▄▄▄▄▄▄▄▄▄▄▄       {cyan}  {blue}Memory{reset}        {memory_usage}
-    {blue}   ▝  {cyan}▟█▜█▖{blue}▀▀▀▀▀██▛▀▀▘       {cyan}󱥎  {blue}Storage (/){reset}   {storage}
-    {cyan}     ▟█▘ ▜█▖    {blue}▝█▛          {cyan}  {blue}Colors{reset}        {colors}");
+    {blue}  ▗▄   {cyan}▗▄ ▄▖      {reset}┌───────────────────────────────────┐
+    {blue} ▄▄🬸█▄▄▄{cyan}🬸█▛ {blue}▃
+    {cyan}   ▟▛    ▜{blue}▃▟🬕       {cyan}Disk: {reset}{storage}
+    {cyan}🬋🬋🬫█      {blue}█🬛🬋🬋      {cyan}RAM: {reset}{memory_usage}
+    {cyan} 🬷▛🮃{blue}▙    ▟▛         {cyan}Uptime: {reset}{uptime}
+    {cyan} 🮃{blue} ▟█🬴{cyan}▀▀▀█🬴▀▀
+    {blue}  ▝▀ ▀▘   {cyan}▀▘      {reset}└───────────────────────────────────┘");
 
     std::io::stdout()
         .lock()
