diff --git a/src/main.rs b/src/main.rs
index 5cb9904..88b29a3 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -67,17 +67,14 @@ fn print_system_info(fields: &Fields) {
         storage,
         colors,
     } = fields;
+    let _ = std::io::stdout().write_all(format!("
+{BLUE}  ▗▄   {CYAN}▗▄ ▄▖      {RESET}┌───────────────────────────────────┐
+{BLUE} ▄▄🬸█▄▄▄{CYAN}🬸█▛ {BLUE}▃       
+{CYAN}   ▟▛    ▜{BLUE}▃▟🬕       {CYAN}Disk: {RESET}{storage}
+{CYAN}🬋🬋🬫█      {BLUE}█🬛🬋🬋      {CYAN}RAM: {RESET}{memory_usage}
+{CYAN} 🬷▛🮃{BLUE}▙    ▟▛         {CYAN}Uptime: {RESET}{uptime}
+{CYAN} 🮃{BLUE} ▟█🬴{CYAN}▀▀▀█🬴▀▀       
+{BLUE}  ▝▀ ▀▘   {CYAN}▀▘      {RESET}└───────────────────────────────────┘
 
-    let _ = std::io::stdout().write_all(format!(
-        "
- {CYAN}     ▟█▖    {BLUE}▝█▙ ▗█▛          {user_info} ~{RESET}
- {CYAN}  ▗▄▄▟██▄▄▄▄▄{BLUE}▝█▙█▛  {CYAN}▖        {CYAN}  {BLUE}System{RESET}        {os_name}
- {CYAN}  ▀▀▀▀▀▀▀▀▀▀▀▘{BLUE}▝██  {CYAN}▟█▖       {CYAN}  {BLUE}Kernel{RESET}        {kernel_version}
- {BLUE}     ▟█▛       {BLUE}▝█▘{CYAN}▟█▛        {CYAN}  {BLUE}Shell{RESET}         {shell}
- {BLUE}▟█████▛          {CYAN}▟█████▛     {CYAN}  {BLUE}Uptime{RESET}        {uptime}
- {BLUE}   ▟█▛{CYAN}▗█▖       {CYAN}▟█▛          {CYAN}  {BLUE}Desktop{RESET}       {desktop}
- {BLUE}  ▝█▛  {CYAN}██▖{BLUE}▗▄▄▄▄▄▄▄▄▄▄▄       {CYAN}󰍛  {BLUE}Memory{RESET}        {memory_usage}
- {BLUE}   ▝  {CYAN}▟█▜█▖{BLUE}▀▀▀▀▀██▛▀▀▘       {CYAN}󱥎  {BLUE}Storage (/){RESET}   {storage}
- {CYAN}     ▟█▘ ▜█▖    {BLUE}▝█▛          {CYAN}  {BLUE}Colors{RESET}        {colors}
-").as_bytes());
+  ").as_bytes());
 }
