diff --git a/src/main.rs b/src/main.rs
index 5cb9904..21f4e86 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -68,16 +68,14 @@ fn print_system_info(fields: &Fields) {
         colors,
     } = fields;
 
-    let _ = std::io::stdout().write_all(format!(
-        "
- {CYAN}     ▟█▖    {BLUE}▝█▙ ▗█▛          {user_info} ~{RESET}
- {CYAN}  ▗▄▄▟██▄▄▄▄▄{BLUE}▝█▙█▛  {CYAN}▖        {CYAN}  {BLUE}System{RESET}        {os_name}
- {CYAN}  ▀▀▀▀▀▀▀▀▀▀▀▘{BLUE}▝██  {CYAN}▟█▖       {CYAN}  {BLUE}Kernel{RESET}        {kernel_version}
- {BLUE}     ▟█▛       {BLUE}▝█▘{CYAN}▟█▛        {CYAN}  {BLUE}Shell{RESET}         {shell}
- {BLUE}▟█████▛          {CYAN}▟█████▛     {CYAN}  {BLUE}Uptime{RESET}        {uptime}
- {BLUE}   ▟█▛{CYAN}▗█▖       {CYAN}▟█▛          {CYAN}  {BLUE}Desktop{RESET}       {desktop}
- {BLUE}  ▝█▛  {CYAN}██▖{BLUE}▗▄▄▄▄▄▄▄▄▄▄▄       {CYAN}󰍛  {BLUE}Memory{RESET}        {memory_usage}
- {BLUE}   ▝  {CYAN}▟█▜█▖{BLUE}▀▀▀▀▀██▛▀▀▘       {CYAN}󱥎  {BLUE}Storage (/){RESET}   {storage}
- {CYAN}     ▟█▘ ▜█▖    {BLUE}▝█▛          {CYAN}  {BLUE}Colors{RESET}        {colors}
-").as_bytes());
+    let _ = std::io::stdout().write_all(format!("
+  ▗▄   ▗▄ ▄▖      ┌───────────────────────────────────┐
+ ▄▄🬸█▄▄▄🬸█▛ ▃       
+   ▟▛    ▜▃▟🬕       Disk: {storage}
+🬋🬋🬫█      █🬛🬋🬋      RAM: {memory_usage}
+ 🬷▛🮃▙    ▟▛         Uptime: {uptime}
+ 🮃 ▟█🬴▀▀▀█🬴▀▀       
+  ▝▀ ▀▘   ▀▘      └───────────────────────────────────┘
+
+  ").as_bytes());
 }
